checkpoint/recovery.go

```diff
@@ -331,7 +332,7 @@ func collectOwnAtxDeps(
 	}
 
 	// check for if miner is building any atx
-	nipostCh, _ := activation.LoadNipostChallenge(cfg.PostDataDir)
+	nipostCh, _ := activation.LoadNipostChallenge(cfg.PostDataDir) // TODO(mafa): fetch from DB instead
```

Comment:
TODO: update checkpoint to read from local state db instead of reading files from disk